boot/bootutil/src/loader.c

```diff
@@ -781,9 +783,20 @@ boot_validate_slot(struct boot_loader_state *state, int slot,
 #if defined(MCUBOOT_OVERWRITE_ONLY) && defined(MCUBOOT_DOWNGRADE_PREVENTION)
     if (slot != BOOT_PRIMARY_SLOT) {
         /* Check if version of secondary slot is sufficient */
-        rc = boot_version_cmp(
-                &boot_img_hdr(state, BOOT_SECONDARY_SLOT)->ih_ver,
-                &boot_img_hdr(state, BOOT_PRIMARY_SLOT)->ih_ver);
+
+	#if defined(CONFIG_SOC_NRF5340_CPUAPP)
```

Comment:
align the # lines to the left and this also needs to check for PCD support, just because an nrf5340 has 2 updateable images doesn't mean the second is for the app core

Reply:
Sure I'll check if NRF53_MULTI_IMAGE and PCD is enabled.

I don't really know how this can be solved more generally as we don't have any extra metadata in the images that indicates that this is for network core currently.

So this is how Successful/Unsuccessful updates for network core will look.
![image](https://github.com/nrfconnect/sdk-mcuboot/assets/3646850/037f8590-54a2-4fff-b972-9972ccf3200c)

Reply:
Fixed alignments. So what will happen if the customer uploads app to slot 1 and network core to slot 0.

I'll do the same trick as the other functions are doing:
using the reset address and compare it against the base address of CPUNET